command/operator_raft_info.go

```diff
@@ -0,0 +1,66 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/hashicorp/nomad/helper/raftutil"
+	"github.com/posener/complete"
+)
+
+type OperatorRaftInfoCommand struct {
+	Meta
+}
+
+func (c *OperatorRaftInfoCommand) Help() string {
+	helpText := `
+Usage: nomad operator raft _info <path to nomad data dir>
+
+  Displays info about the raft logs in the data directory
```

Comment:
I'd like to put some big warning flags in here to give us a much leeway as possible to change these commands at our discretion without worrying about our usual compatibility guarantees. Hopefully we can also hint at the lack of testing and polish with a phrase like "low-level debugging tool". I'm not too worried about the wording. I just want us to make sure we're (a) setting the right expectations with users and (b) giving ourselves freedom to change this operations as suites our needs.

```suggestion
  Displays info about the raft logs in the data directory.
  
  This is a low-level debugging tool and not subject to Nomad's usual backward
  compatibility guarantees.
```